Tasks/Common/PowershellHelpers/Tests/Invoke-ActionWithRetries.ShouldNotRetryInvalidExceptionType.ps1

```diff
@@ -0,0 +1,21 @@
+[CmdletBinding()]
+param()
+
+# Arrange.
+. $PSScriptRoot\..\..\..\..\Tests\lib\Initialize-Test.ps1
+$module = Microsoft.PowerShell.Core\Import-Module $PSScriptRoot\.. -PassThru
+$global:retriesAttempted = 0
+$action = {
+    $global:retriesAttempted++
+    throw "Invalid error!"
+}
+
+Register-Mock Set-UserAgent
+Unregister-Mock Start-Sleep
+Register-Mock Start-Sleep {}
+
+# Act/Assert.
+Assert-Throws {
+    & $module Invoke-ActionWithRetries -Action $action -RetryableException "System.Exception"
+} -MessagePattern "Invalid error!"
+Assert-AreEqual 1 $global:retriesAttempted "Number of retries not correct"
```

Comment:
This was a issue with test which was not updated after change to match assignable exception. Fixed test now.